app/hooks/useChat.ts

```
@@ -0,0 +1,185 @@
+    // If all checks pass, send the message
+    const result = chatManagerRef.current.sendMessage(content);
+    if (!result) {
+      logger.error('Failed to send message, updating chat ready state');
+      setChatReady(false);
+    }
+    
+    return result;
+  }, [logger]);
+
+  // Wait for the chat data channel to be ready
+  const waitForReady = useCallback(async (timeout: number = 5000) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)